infra/base-images/base-runner/coverage

```diff
@@ -23,7 +23,10 @@ else
 fi
 
 LOGS_DIR="$OUT/logs"
-mkdir -p $LOGS_DIR
```

Comment:
Would it be worth deleting the entire $OUT directory?

Reply:
No, there must be fuzzer binaries which are needed for generating code coverage report.